### PR TITLE
add ros package

### PR DESCRIPTION
**Pull request by 885288421**
Please add the following dependency to the rosdep database.

## Package name:

agxrobots_test

## Package Upstream Source:
[
TODO link to source repository](https://github.com/885288421/agxrobots_test_release.git)

## Purpose of using this:

this software package provides a C++ interface to communicate with the mobile platforms from Weston Robot and AgileX Robotics, for sending commands to the robot and receiving the latest robot state.

Distro packaging links:
https://github.com/885288421/agxrobots_test_release.git

More info at https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-rosdep-rules -->

# Please Add This Package to be indexed in the rosdistro.

ROSDISTRO NAME

# The source is here: 

https://github.com/885288421/agxrobots_test.git

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -149,6 +149,35 @@ repositories:
       url: https://github.com/ubi-agni/agni_tf_tools.git
       version: master
     status: maintained
+  agxrobots_test:
+    doc:
+      type: git
+      url: https://github.com/885288421/agxrobots_test.git
+      version: 0.0.3
```
Comment (tfoote) [inline melodic/distribution.yaml:156]:
```suggestion
      version: master
```

This should be a branch name.

File: melodic/distribution.yaml [hunk 2]
```diff
@@ -149,6 +149,35 @@ repositories:
...
+      - tracer_msgs
+      - ugv_sdk
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/885288421/agxrobots_test_release.git
+      version: 0.0.3-8
+    source:
+      type: git
+      url: https://github.com/885288421/agxrobots_test.git
+      version: 0.0.3
```
Comment (tfoote) [inline melodic/distribution.yaml:179]:
```suggestion
      version: master
```
This should be a branch name.

**Review by tfoote — CHANGES_REQUESTED** [review]
The branch names need to be updated. 

I also note that this is coming from what appears to be forks in your user's account versus the upstream packages on agilexrobotics which is a little surprising.

**Comment by tfoote** [comment]
Related: #31625

**Comment by 885288421** [comment]
I will pull my all code to the agilextobotices,and I will change the all url.



At 2022-01-12 10:37:57, "Tully Foote" ***@***.***> wrote:

@tfoote requested changes on this pull request.

The branch names need to be updated.

I also note that this is coming from what appears to be forks in your user's account versus the upstream packages on agilexrobotics which is a little surprising.

In melodic/distribution.yaml:

> @@ -149,6 +149,35 @@ repositories:

       url: https://github.com/ubi-agni/agni_tf_tools.git

       version: master

     status: maintained

+  agxrobots_test:

+    doc:

+      type: git

+      url: https://github.com/885288421/agxrobots_test.git

+      version: 0.0.3


⬇️ Suggested change
-      version: 0.0.3

+      version: master



This should be a branch name.

In melodic/distribution.yaml:

> +      - scout_bringup

+      - scout_msgs

+      - tracer_base

+      - tracer_bringup

+      - tracer_description

+      - tracer_gazebo_sim

+      - tracer_msgs

+      - ugv_sdk

+      tags:

+        release: release/melodic/{package}/{version}

+      url: https://github.com/885288421/agxrobots_test_release.git

+      version: 0.0.3-8

+    source:

+      type: git

+      url: https://github.com/885288421/agxrobots_test.git

+      version: 0.0.3


⬇️ Suggested change
-      version: 0.0.3

+      version: master



This should be a branch name.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by wjwwood** [comment]
> I will pull my all code to the agilextobotices,and I will change the all url.

I think we're still waiting on this?

**Comment by tfoote** [comment]
@885288421 When do you plan to move the repositories? Since this is 3 weeks old. I will plan to close it. When you move them please reopen or submit a new pull request with the updated urls.

**Pull request closed without merging**